Userland/DevTools/HackStudio/ProjectTemplate.cpp

```diff
@@ -76,8 +70,10 @@ Result<void, String> ProjectTemplate::create_project(const String& name, const S
     // Verify that the template content directory exists. If it does, copy it's contents.
     // Otherwise, create an empty directory at the project path.
     if (Core::File::is_directory(content_path())) {
-        if (!copy_directory(content_path(), path, false))
-            return String("Failed to copy template contents.");
+        auto result = Core::File::copy_file_or_directory(path, content_path());
```

Comment:
not sure I agree with the choice to swap src and dest here, but that's not your fault of course :smile:

Reply:
Yeah, threw me off for a bit as well.